### ISSUE TITLE
Windows CI build failures

### ISSUE THREAD

**Issue by slouken** [libsdl-org/SDL#9714]
The Windows CI is failing with:
```
Installing additional packages through pacman...
  C:\Windows\system32\cmd.exe /D /S /C D:\a\_temp\setup-msys2\msys2.cmd -c "'pacman' '--noconfirm' '-S' '--needed' '--overwrite' '*' 'mingw-w64-clang-i686-cc' 'mingw-w64-clang-i686-cmake' 'mingw-w64-clang-i686-ninja' 'mingw-w64-clang-i686-perl' 'mingw-w64-clang-i686-pkg-config' 'mingw-w64-clang-i686-clang-tools-extra'"
  error: target not found: mingw-w64-clang-i686-perl
```

@madebr, can you take a look?

**Comment by madebr** [comment]
I'll ask around on msys2 channels.

**Comment by madebr** [comment]
Fixed in 818721fc9 by not building the docs for mingw32/clang32

Perl is not built anymore for 32-bit platforms since https://github.com/msys2/MINGW-packages/pull/20085

**Comment by slouken** [comment]
Thanks!
